### PR TITLE
coordinator: Restore downloads snapshots from requested storage

### PR DESCRIPTION
**Pull request by ilpon**

### CODE REVIEW

File: tests/unit/coordinator/test_restore.py [hunk 1]
```diff
@@ -41,9 +42,11 @@
 )
 
 
+@pytest.mark.parametrize("storage_name", [None, "x", "y"])
```
Comment (fingon) [inline tests/unit/coordinator/test_restore.py:45]: This leads to 3 * 4 = 12 tests. I don't think all of the combinations are really useful?

**Review by fingon — DISMISSED** [review]
LGTM, test could use some refactoring though as adding 9 more tests due to this feels excessive and they are pretty duplicate ones too.

**Review by fingon — APPROVED** [review]
